### ISSUE TITLE
GO:0006139	 nucleobase-containing compound metabolic process IEA with ARBA00026572

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5461]
* **ARBA rule (ID and label):**

GO:0006139 | nucleobase-containing compound metabolic process | IEA with ARBA00026572

* **Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPCC4G3.16

* **Description of issue**
* 
rib2 is involved in riboflavin biosynthesis
Riboflavin lacks the nitrogenous base component that defines nucleotides.

**Comment by Antonialock** [comment]
@ValWood I see the cerevisiae ortholog is annotated to "trna pseudouridine synthesis" IMP + IDA, so this looks ok?

![Image](https://github.com/user-attachments/assets/ae47d8c6-8858-47d1-bea5-164cd5df1b20)

**Comment by ValWood** [comment]
Unfortunatly not, this is a weird situation.

In S. cerevisiae this is a tandem fusion (RIB2 plus PUS9 paralog), and S. pombe is a tandem fusion (which maps to RIB2 and RKM5),

but the only common part between pombe RIB2 and S. cerevsiaie RIB2 is the 
diaminohydroxyphosphoribosylaminopyrimidine deaminase activity/ riboflavin biosynthesis

It might be useful for ARBA to have exceptions for tandem fusions.
We had to do this with the MAtrix co-annotation project PMID: 32875947, so we already have some rules captured for these cases. Although there are not many so far, I could dig these out....

**Comment by Antonialock** [comment]
Maybe the issue comes from interpro IPR002125 Cytidine and deoxycytidylate deaminase domain 

I see that pombe rib2 is not annotated to those two activities, but instead to the sibling term "diaminohydroxyphosphoribosylaminopyrimidine deaminase activity"

![Image](https://github.com/user-attachments/assets/0c3b2760-ac43-4583-ab3f-9a7b6c766737)

when I look at the co-annotated processes of cytidine and deoxycytidylate deaminase activity, then the most common co-annotated process terms all have links to nucleobase-containing compound metabolism.

i.e. not all protiens with this domain have cytidine ordeoxycytidylate deaminase activity

![Image](https://github.com/user-attachments/assets/229b8986-6b08-4c9f-abe7-0df5a53b9ea8)

**Comment by Antonialock** [comment]
the ARBA rule page (https://www.uniprot.org/arba/ARBA00026572) says https://www.ebi.ac.uk/interpro/entry/InterPro/IPR016193/ Cytidine deaminase-like for fungi

Will propagation be blocked for this interpro if you put in a NOT? (I don't know if ARBA takes NOTs into account?)

**Comment by ValWood** [comment]
but IPR002125 doesn't have a mapping to any MF terms (probably because it can't be mapped unambiguously)

**Comment by ValWood** [comment]
I mean https://www.ebi.ac.uk/interpro/entry/InterPro/IPR016193/

**Comment by ValWood** [comment]
can't be mapped because it's a superfamily
https://www.pombase.org/gene_subset/interpro:IPR016193

**Comment by ValWood** [comment]
The 
GO:0006139 nucleobase-containing compound metabolic process
comes from  the pseudouridine synthase activity. This activity/domain isn't present in the pombe protein. i

**Comment by Antonialock** [comment]
> But NEITHER of these is a cytidine deaminase.

 no, but are they annotated to nucleobase-containing compound metabolic process?
I was just trying to understand how the rule was created, and I assumed that the annotated things with this domain, were annotated to "nucleobase-containing compound metabolic process"

**Comment by ValWood** [comment]
note that there are many proteins which have only the DHAP deaminase activity, not fused to the pseudouridine synthase activity

<img width="484" alt="Image" src="https://github.com/user-attachments/assets/46a0a643-1bcf-452b-93ff-d2bfe1c226d2" />

so fission yeast RIB2 won't be the only affected  protein. 

But I can't see exactly where the  ARBA inference is coming from.
I'm guessing that what is happening is that many proteins have 
pseudouridine synthase - and  a BP with a  nucleotide containing-compound parentage (includes S. cerevisiae)
Many proteins have cytidine deaminase  and a process with a BP nucleotide containing-compound parentage (excludes pombe and S. cerevisiae RIB2)

So there is a high probability that the common parent is nucleotide containing-compound parentage,
but not for organisms where the DRAP deaminase isn't fused to the pseudouridine synthase

**Comment by Antonialock** [comment]
this term has been made do_not_annotate and the annotation is gone in p2GO
